### PR TITLE
[zh-cn]: update the translation of URL `host` property 

### PR DESCRIPTION
**Pull request by T34-active**
### Description
and URL `hash` property add sourceCommit
https://developer.mozilla.org/en-US/docs/Web/API/URL/host
https://developer.mozilla.org/en-US/docs/Web/API/URL/hash

### CODE REVIEW

File: files/zh-cn/web/api/url/host/index.md [hunk 1]
```diff
@@ -1,41 +1,30 @@
 ---
-title: URL.host
+title: URL：host 属性
 slug: Web/API/URL/host
+l10n:
+  sourceCommit:
```
Comment (T34-active) [inline files/zh-cn/web/api/url/host/index.md:5]:
```suggestion
  sourceCommit: 1a91b0b63f0cbaca9125bd48d4e5bc8afed2a7a3
```

File: files/zh-cn/web/api/url/hash/index.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 title: URL：hash 属性
 slug: Web/API/URL/hash
+l10n:
+  sourceCommit: 4de6f76bbfd76229db78ffb7d52cf6b4cb9f31f8
 ---
 
-{{ APIRef("URL API") }}
+{{ APIRef("URL API") }} {{AvailableInWorkers}}
 
 {{domxref("URL")}} 接口的 **`hash`** 属性是一个包含 URL 标识中的 `'#'` 和片段（fragment）标识符的字符串。
```
Comment (yin1999) [inline files/zh-cn/web/api/url/hash/index.md:10]: 还请麻烦更新一下这句的翻译

File: files/zh-cn/web/api/url/host/index.md [hunk 3]
```diff
@@ -1,41 +1,30 @@
...
 slug: Web/API/URL/host
+l10n:
+  sourceCommit: 1a91b0b63f0cbaca9125bd48d4e5bc8afed2a7a3
 ---
 
-{{ApiRef("URL API")}}
+{{ApiRef("URL API")}} {{AvailableInWorkers}}
 
-{{domxref("URL")}} 接口的 **`host`** 属性是一个 {{domxref("USVString")}} 值，包含了主机信息，也就是 _主机名（hostname）_，还有，如果 URL 接口不为空，也会包含冒号 `':'` 和 URL 的 _端口（port）_。
+{{domxref("URL")}} 接口的 **`host`** 属性是一个包含主机信息的字符串，即{{domxref("URL.hostname", "主机名", "", 1)}}，然后，如果 URL 的{{glossary("port", "端口")}}非空，则会出现 `':'`，后面是 URL 的{{domxref("URL.port", "端口", "", 1)}}。
```
Comment (yin1999) [inline files/zh-cn/web/api/url/host/index.md:10]:
```suggestion
{{domxref("URL")}} 接口的 **`host`** 属性是一个包含主机信息（即{{domxref("URL.hostname", "主机名", "", 1)}}）的字符串，然后，如果 URL 的{{glossary("port", "端口")}}非空，则会出现 `':'`，后根 URL 的{{domxref("URL.port", "端口", "", 1)}}。
```

File: files/zh-cn/web/api/url/hash/index.md (hunk 2 above)
Comment (T34-active) [inline files/zh-cn/web/api/url/hash/index.md:10]:
```suggestion
{{domxref("URL")}} 接口的 **`hash`** 属性是一个包含以 '#' 开头的 URL 片段标识符的字符串。
```

**Review by yin1999 — APPROVED** [review]
感谢 @T34-active

**Pull request merged**